crates/e2e/sandbox/src/macros.rs

```diff
@@ -193,6 +193,8 @@ mod construct_runtime {
         type MaxCodeLen = ConstU32<{ 123 * 1024 }>;
         type MaxStorageKeyLen = ConstU32<128>;
         type UnsafeUnstableInterface = ConstBool<false>;
+        type UploadOrigin = $crate::frame_system::EnsureSigned<Self::AccountId>;
+    type InstantiateOrigin = $crate::frame_system::EnsureSigned<Self::AccountId>;
```

Comment:
```suggestion
        type InstantiateOrigin = $crate::frame_system::EnsureSigned<Self::AccountId>;
```
one more time :-) 8 spaces

Reply:
It looks okay to me:

![image](https://github.com/paritytech/ink/assets/75586/99642c5a-cf7a-487e-8df0-153b2491f950)